src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    private Movie movie;
+
+    @OneToOne
```

Comment:
One cinema hall can have many movie sessions.
```suggestion
    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    @JoinColumn(name = "cinema_hall_id", nullable = false)
```

Reply:
That's true. Changed.